assets/chat/js/regex.js

```diff
@@ -1,10 +1,13 @@
+import { UrlFormatter } from './formatters';
+
 const regexslashcmd = /^\/([a-z0-9]+)[\s]?/i;
 const regextime =
   /(?<number>\d+)(?<unit>s(econds?)?$|m(inutes?)?$|h(ours?)?$|d(ays?)?$)?/i;
 const nickmessageregex = /(?=@?)(\w{3,20})/g;
 const nickregex = /^\w{3,20}$/;
 const nsfwregex = /\bNSFW\b/i;
 const nsflregex = /\bNSFL\b/i;
+const { linkregex } = new UrlFormatter();
```

Comment:
Let's instead move the link regex from `UrlFormatter` to `regex.js` and import it in `UrlFormatter`. That makes a little more sense to me.